db/migrate/20221217121221_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.string :text
```

Comment:
- [ ] Same remark about the text I think the text should be `text` and not `string` 🙃, please kindly fix that 😉